orchestrator-salt/src/main/resources/salt/salt/top.sls

```diff
@@ -11,6 +11,7 @@ base:
     - metering
     - ntp
     - postgresql.root-certs
+    - ccm
```

Comment:
I think we should add the script only if CCM is enabled, so the state should be role dependent. Also only gateway nodes configured for CCM if I remember well, so we should not add this to every node